### ISSUE TITLE
Custom checkbox is broken with empty label

### ISSUE THREAD

**Issue by syabro** [reactstrap/reactstrap#1320]
- components: `CustomInput`
- reactstrap version `#6.5.0`
- import method `es`
- react version `#16.5.2`
- bootstrap version `#4.1.3`

### What is happening?

If label is `""` (empty string) or `undefined` layout is broken

![image](https://user-images.githubusercontent.com/29029/49800227-07866e00-fd58-11e8-95c8-e035e55f03db.png)

### What should be happening?

Checkbox itself should be rendered correctly

![image](https://user-images.githubusercontent.com/29029/49800260-1d942e80-fd58-11e8-8d28-0f2e8c2a2b30.png)

### Error message in console

set label="", f.e.
```
<CustomInput
    id="test"
    type="checkbox"
    label=""
/>
```

### Video example
http://s.syabro.com/2018-12-11_15-14-57.mp4

Maybe replace label rendering with:
```
<label className={mapToCssModules('custom-control-label', cssModule)} htmlFor={attributes.id}>{(label === '' || label === undefined) ? '&nbsp;' : label}</label>
```

**Comment by TheSharpieOne** [comment]
This is an issue with bootstrap itself and has already been fixed (it will be released in bootstrap v4.2.0)

See https://github.com/twbs/bootstrap/pull/27566

Current workaround: Add this style to your css:
```css
.custom-control-label {
  vertical-align: top;
}
```

Demo: https://stackblitz.com/edit/reactstrap-v6-7z5g6j?file=style.css

**Comment by syabro** [comment]
@TheSharpieOne thanks
